src/components/views/right_panel/UserInfo.js

```diff
@@ -1306,7 +1306,7 @@ const BasicUserInfo = ({room, member, groupId, devices, isRoomEncrypted}) => {
 
     const showDeviceListSpinner = devices === undefined;
     if (canVerify) {
-        if (hasCrossSigningKeys !== undefined) {
+        if (hasCrossSigningKeys !== undefined && devices.length > 0) {
```

Comment:
Perhaps it belongs as part of the `canVerify` chain, since that's the purpose you are trying to represent?